vars/Debian-8.yml

```diff
@@ -21,7 +21,7 @@ build_dependencies:
   - libgcc1      # from package dependencies
   - libc6        # from package dependencies
   - libacl1      # from package dependencies
-  - libssl1.0.0  # from package dependencies
+  - libssl1.1    # from package dependencies (debian9,debian10,Ubuntu18+)
```

Comment:
Debian-8 will not work with this library

Reply:
ok! true.